x-pack/legacy/plugins/code/server/routes/search.ts

```diff
@@ -9,12 +9,16 @@ import Boom from 'boom';
 import { RequestFacade, RequestQueryFacade } from '../../';
```

Comment:
for the `'/api/code/integration/snippets'` route in this file, you also need to apply the scope as well. 

Insert the `getScope()` between line 189 and 190 to rewrite the `repoUris` field of the `stacktraceReq: StackTraceSnippetsRequest`. 
https://github.com/elastic/kibana/blob/c51b39ce99c1cde2cbcf12bb5a040749bca1344b/x-pack/legacy/plugins/code/server/routes/search.ts#L189-L191

Reply:
Fixed